cmake/onnxruntime_unittests.cmake

```diff
@@ -143,12 +143,19 @@ file(GLOB onnxruntime_test_utils_src CONFIGURE_DEPENDS
   "${TEST_SRC_DIR}/util/*.cc"
 )
 
-file(GLOB onnxruntime_test_common_src CONFIGURE_DEPENDS
-  "${TEST_SRC_DIR}/common/*.cc"
-  "${TEST_SRC_DIR}/common/*.h"
-  "${TEST_SRC_DIR}/common/logging/*.cc"
-  "${TEST_SRC_DIR}/common/logging/*.h"
+if (NOT onnxruntime_BUILD_WEBASSEMBLY)
+  file(GLOB onnxruntime_test_common_src CONFIGURE_DEPENDS
+    "${TEST_SRC_DIR}/common/*.cc"
+    "${TEST_SRC_DIR}/common/*.h"
+    "${TEST_SRC_DIR}/common/logging/*.cc"
```

Comment:
Is it possible to add unit tests for logging?

Reply:
It's due to pthread_self() call in googletest. I'll disable pthread from googletest build.